resources/extruders/bondtech_bmgx2m_extruder_0.def.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Did you intend to make this for the Bondtech extruder kits?

Reply:
Yes, I made that definition for the Bondtech extruders as that is what I am using, but I can easily configure that based on the custom extruder definitions when setting up the printer profile. 

Deleted these files.